near-sdk/src/environment/env.rs

```diff
@@ -500,6 +500,8 @@ pub fn alt_bn128_pairing_check(value: &[u8]) -> bool {
 // ################
 /// Creates a promise that will execute a method on account with given arguments and attaches
 /// the given amount and gas.
```

Comment:
@PolyProgrammist suggest adding more links, mentioned in #1265 and marked up in review of #1259, 
in addition to ones, added in scope of current pr, after fixing conflicts, if you plan to finish current pr

Reply:
Which links do you mean? @dj8yfo

Reply:
Added half of the comments. Left the other for the future